src/MetaMorpho.sol

```diff
@@ -355,6 +355,7 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
                 }
 
                 delete config[id];
+                delete pendingCap[id];
```

Comment:
[optional] I realize now that these changes (both `config[id]` and `pendingCap[id]`) may be difficult to follow onchain with events. This requires to do the difference of sets to know which markets were removed. Maybe emitting an event here would be nice for offchain purposes

Reply:
cc @julien-devatom